subprojects/docs/src/docs/release/notes.md

```diff
@@ -82,96 +82,103 @@ vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv -->
 
 #### Added support for incremental compilation following a compilation failure
 
-Gradle already supports [Java incremental compilation](userguide/java_plugin.html#sec:incremental_compile) by default and [Groovy incremental compilation](userguide/groovy_plugin.html#sec:incremental_groovy_compilation) as an opt-in experimental feature.
-
-In previous versions after a compilation failure the next compilation was not incremental but a full recompilation instead.
-
-With this version, Java and Groovy incremental compilation will work incrementally also after a failure.
-
-This improves experience with compilation when working iteratively on some Java or Groovy code, e.g. when iteratively running compile or test tasks from an IDE.
+Gradle supports [Java incremental compilation](userguide/java_plugin.html#sec:incremental_compile)
+by default and [Groovy incremental compilation](userguide/groovy_plugin.html#sec:incremental_groovy_compilation)
+as an opt-in experimental feature.
```

Comment:
@asodja This should probably at the end be synchronized with your changes in https://github.com/gradle/gradle/pull/21878

Reply:
Thanks for pointing that out. I believe so yes.